src/firefly/js/tables/reducer/TableUiReducer.js

```diff
@@ -79,7 +73,21 @@ function uiStateReducer(ui, tableModel) {
         ui = update(ui, {$merge: {[tbl_ui_id]: uiData}});
     });
     return ui;
+}
 
+function updateAllUi(ui, tbl_id, tbl_ui_id, payload) {
+    if (tbl_ui_id) {
+        const changes = set({}, [tbl_ui_id], payload);
+        return TblUtil.smartMerge(ui, changes);
```

Comment:
concerning my previous comment about update would a `set`/`smartMerge` be a substitute for using `update` with a `$set`?